docs/docs/segment-az.md

```diff
@@ -31,3 +31,27 @@ Display the currently active Azure subscription information.
 - info_separator: `string` - text/icon to put in between the subscription name and ID - defaults to ` | `
 - display_id: `boolean` - display the subscription ID or not - defaults to `false`
 - display_name: `boolean` - display the subscription name or not - defaults to `true`
+
+## Az PowerShell Module
+
+By default, the az segment will retrieve subscription information for the [Azure CLI][az-cli], not the [Az PowerShell Module][pwsh-module].
+If you use the Az PowerShell Module, add the code below to your PowerShell profile. This will cause
+the az segment to show information relevant to the Az PowerShell module.
+
+```pwsh
+function Set-EnvVar {
+    try {
+        $subscription = Get-AzContext | Select-Object -ExpandProperty "Subscription" | Select-Object "Name", "Id"
```

Comment:
we could preload this just like we do for `posh-git`. If that cmdlet is available, add those env vars.

Reply:
done!